src/consistency-testing/chaostest/test-plan-templates/kafka.comrmw.ext.json

```diff
@@ -0,0 +1,213 @@
+            "create_topic_script": "/opt/kafka_2.12-2.6.0/bin/kafka-topics.sh --create --topic topic1 --partitions 1 --replication-factor 3 --bootstrap-server 427.0.0.3:9092",
+            "strobe_start_api_script": "sudo -b nohup python3 /home/ubuntu/control/strobe.py --port 9094 --storage /home/ubuntu/strobe.json --log /home/ubuntu/strobe.log > /home/ubuntu/strobe.out.log 2>&1 < /dev/null &",
+            "strobe_kill_api_script": "sudo /home/ubuntu/control/kafka/strobe.kill.sh",
+            "strobe_recover_script": "sudo /home/ubuntu/control/kafka/strobe.recover.sh",
+            "strobe_inject_script": "sudo /home/ubuntu/control/kafka/strobe.inject.sh 200000 1000",
+            "meta_script": "kafkacat -L -b 427.0.0.3 -t topic1 -J",
+            "start_zookeeper_script": "sudo -b nohup /opt/apache-zookeeper-3.6.2-bin/bin/zkServer.sh start-foreground > /home/ubuntu/zookeeper.log 2>&1 < /dev/null &",
+            "start_kafka_script": "sudo -b nohup /opt/kafka_2.12-2.6.0/bin/kafka-server-start.sh /opt/kafka_2.12-2.6.0/config/server.properties > /home/ubuntu/kafka.log 2>&1 < /dev/null &",
+            "kill_kafka_script": "sudo /home/ubuntu/control/kafka/kill-kafka.sh",
+            "kill_zookeeper_script": "sudo /home/ubuntu/control/kafka/kill-zookeeper.sh",
+            "zookeeper_config": {
+                "path": "/opt/apache-zookeeper-3.6.2-bin/conf/zoo.cfg",
+                "dataDir": "/mnt/vectorized/front/zookeeper",
+                "server1": "427.0.0.1:2888:3888",
+                "server2": "427.0.0.2:2888:3888",
+                "server3": "0.0.0.0:2888:3888",
+                "id": 3
+            },
+            "kafka_config": {
+                "path": "/opt/kafka_2.12-2.6.0/config/server.properties",
+                "logDirs": "/mnt/vectorized/front/kafka",
+                "zookeeper": "427.0.0.1:2181,427.0.0.2:2181,427.0.0.3:2181",
+                "ip": "427.0.0.3:9092",
+                "id": 6
+            }
+        }
+    ],
+    "workload": {
+        "name": "comrmw",
+        "period_s": 0.01
+    },
+    "system": "kafka",
+    "scenario": "inject-recover",
+    "exit_on_violation": false,
+    "reset_before_test": true,
+    "destroy_on_exit": true,
+    "output": "kafka",
+    "writers": 3,
+    "readers": 3,
+    "warmup": 60,
+    "verbose": true,
+    "cluster_warmup": 30,
+    "exploitation": 60,
+    "cooldown": 120,
+    "ss_metrics": ["send_us", "catchup_us"],
+    "cmd_log": "cmd.log",
+    "latency_log": "lat.log",
+    "availability_log": "1s.log",
+    "faults": [
+        "baseline",
+        "terminate.leader",
+        "terminate.follower",
+        "suspend.leader",
+        "suspend.follower",
+        "isolate.leader",
+        "isolate.follower",
+        "io10ms.leader",
+        "io10ms.follower",
+        "iofail.leader",
+        "iofail.follower",
+        "strobe.leader",
+        "strobe.follower"
+    ]
+}
```

Comment:
missing a newline. if you're using vscode there is a setting that will do this automatically (https://github.com/microsoft/vscode/issues/58276#issuecomment-420660854). i think vim/emacs do by default.

Reply:
added a new line